x-pack/plugins/security_solution/cypress/integration/hosts/events_viewer.spec.ts

```diff
@@ -86,13 +90,10 @@ describe('Events Viewer', () => {
   });
 
   context('Events viewer fields behaviour', () => {
-    before(() => {
+    beforeEach(() => {
```

Comment:
Why is the reason behind moving `cleanKibana` method to ta `beforeEach` hook?  That method is really expensive in time execution and those tests do not modify any of the indexes that it cleans. So it is ok to clean the status just at the beginning of the context execution, in order to save execution time. If there is not a big reason behind that move, I'll keep it on a before hook.

Reply:
There is some sort of bug with this set of tests where not everything that should be is cleaned up between runs, and the test fails, something related to my change of debouncing when the filter is applied, as       filterFieldsBrowser(filterInput) will change the text in the input element, however the list of field checkboxes does not change. A full reload of the browser cleans up whatever state is improperly leaking and the test works as expected, and since every other it block in this file calls cleanKibana already, it didn't seem like a big deal

Reply:
If a full reload of the browser is ok, just move the cleanKibana method to the before hook and keep the rest on the beforeEach since the login and navigation is forcing a refresh.

Reply:
@MadameSheema done, thanks for the suggestion